### DISCUSSION TITLE
Plan for PS7.2+ and .NET 6+ MAUI and CUI tools integrated?

### DISCUSSION THREAD

**Discussion by suglasp** [PowerShell/PowerShell#17441] [Q&A] [answered]
Will Powershell in the future (planned) contain more added value for GUI and CUI development and utilities?
- for example .NET 6 MAUI for cross platform GUI's (https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-6).
- WinUI 3 (I see MAUI more fit personally against what Powershell offers for WinForms and WPF, since it also is the latest Desktop UI platform)
- For CUI's, something like the ConsoleGUI tools integrated into Powershell native (https://www.powershellgallery.com/packages/Microsoft.PowerShell.ConsoleGuiTools/0.6.2)

Before you ask : Yes, I have read Issue #13063. My question here is, more of a extended version of the issue in a question like sense.

**Comment by kasini3000** [comment, marked as answer]
my script:《PowerAlias》, linux + powershell + tui  (base gui.cs lib)
https://gitee.com/chuanjiao10/powershell/tree/master/win-linux%E9%80%9A%E7%94%A8/poweralias

I am looking forward to being able to use powershell v7 scripts to call the maui library and build gui programs on android and ios. Just like building a wpf program on win via powershell.

I also want to use powershell to write a mahjong game, but I haven't started it for 5 years
